locales/en.json

```diff
@@ -92,7 +92,7 @@
   },
   "app_desc_short": "Decentralized social media powered by Mastodon",
   "app_logo": "Mozilla Logo",
-  "app_name": "Mozilla Social",
+  "app_name": "Mozilla.social",
```

Comment:
Went ahead and updated the English app name to match the `Mozilla.social` syntax that product decided on